src/main/java/com/dezorganizacja/bridge/view/text/OrdersHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.dezorganizacja.bridge.view.text;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by maciek on 13.04.15.
+ */
+
+
+interface Command {
+    void runCommand(ArrayList<String> args);
+}
+
+public class OrdersHandler {
+    private Map<String, Command> menuEntries = new HashMap<>();
+    private Map<String, String> menuDetails = new HashMap<>();
+    {
+        menuEntries.put("h", (ArrayList<String> args)->{this.help(args);});
+        menuEntries.put("help", (ArrayList<String> args)->{this.help(args);});
+        menuEntries.put("credits", (ArrayList<String> args)->{this.credits(args);});
+        menuEntries.put("menu", (ArrayList<String> args)->{this.print_menu(args);});
```

Comment:
InteliJ mi ładnie krzyczy: "Can be rapleced with method reference". czyli np this::help